integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -41,15 +43,47 @@
 public final class AlluxioFuse {
   private static final Logger LOG = LoggerFactory.getLogger(AlluxioFuse.class);
 
+  private static final Option MOUNT_POINT_OPTION = Option.builder("m")
```

Comment:
NOTE, this is some cleanup work I planned to do a long time ago. unrelated to @iluoeli 's change to switch to JNI